sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/telemetry/Telemetry.scala

```diff
@@ -60,13 +60,25 @@ final case object EventSourcedEntityCategory extends ComponentCategory {
   def name = "Event Sourced Entity"
 }
 
+final case object ValueEntityCategory extends ComponentCategory {
+  def name = "Value Entity"
+}
+
 final class Telemetry(system: ActorSystem) extends Extension {
 
-  def traceInstrumentation(componentName: String, componentCategory: ComponentCategory) =
-    if (system.settings.config.getString(TraceInstrumentation.TRACING_ENDPOINT).isEmpty)
+  val logger = LoggerFactory.getLogger(classOf[Telemetry])
+
+  def traceInstrumentation(componentName: String, componentCategory: ComponentCategory) = {
+    val collectorEndpoint = system.settings.config.getString(TraceInstrumentation.TRACING_ENDPOINT)
+    if (collectorEndpoint.isEmpty) {
+      logger.info("Instrumentation disabled. Set to NoOp.")
```

Comment:
debug, we do not want these showing up in user logs on every user service component startup (on every node start/restart in deployed app)

Reply:
done